documentation/help-documenting.rst

```diff
@@ -36,11 +36,10 @@ and :ref:`maintenance <maintbranch>` branches at https://docs.python.org/dev/.
 The in-development and recent maintenance branches are rebuilt once per day.
 
 If you would like to be more involved with documentation, consider subscribing
-to the `docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
-mailing list and the `Documentation category on the Python Discourse
-<https://discuss.python.org/c/documentation/26>`_,
-where user issues are raised and documentation toolchain, projects, and standards
-are discussed.
+to the `Documentation category on the Python Discourse
+<https://discuss.python.org/c/documentation/26>`_ and the `docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
+mailing list where user issues are raised and documentation toolchain,
+projects, and standards are discussed.  
```

Comment:
Fix trailing whitespace (causing the lint failure), and rationalize the line breaks to minimize diffs and keep the line lengths at least roughly consistent.

```suggestion
to the `Documentation category on the Python Discourse
<https://discuss.python.org/c/documentation/26>`_ and the
`docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_ mailing list
where user issues are raised and documentation toolchain, projects, and standards
are discussed.
```

Reply:
Hey @Mariatta , do we just want to commit this and merge, or do you have other ideas here? Thanks!

Reply:
Per Mariatta's confirmation, going ahead with this.

Reply:
Because it seems the branch was mistakenly pushed to the upstream repo rather than the user's fork (as the user was an admin, the current protections against this didn't work—I've tested and confirmed an improved version on python/core-workflow#460 that would prevent accidents like this going forward), @Mariatta or one of her fellow admins will have to commit this suggestion.